src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -161,6 +193,32 @@ const ActiveState = ({
                             </ActivityItem>
                         );
                     case 'annotation':
+                        if (hasReplies) {
+                            return (
+                                <ActivityItem
+                                    key={item.type + item.id}
+                                    className="bcs-activity-thread"
+                                    data-testid="activity-thread"
+                                    isFocused={isFocused}
+                                    ref={refValue}
+                                >
+                                    <ActivityThread {...item}>
```

Comment:
Do we need to spread `{...item}` here or can it be omitted?

Reply:
It can be omitted currently and added in next PR 👍